roles/taca/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 taca_ngi_repo: https://github.com/SciLifeLab/taca-ngi-pipeline.git
 taca_ngi_dest: "{{ sw_path }}/taca-ngi-pipeline"
-taca_ngi_version: 1d7ca548099ed3b08dd33fff59f6b8652b323707
+taca_ngi_version: 79d6f1b38dd5808b0ab4a3e7a732c989e5cc7ec1
```

Comment:
This would go in monthly too.

Reply:
This version is available in master and monthly, it is only the bimonthly branch that had an older version.

Reply:
Ah I misunderstood. This is fine!